extras/ams_wrapper/src/api/models/model.py

```diff
@@ -18,39 +18,38 @@
 import json
 import falcon
 import tensorflow as tf
+import sys
 import numpy as np
 from abc import ABC, abstractmethod
 from logger import get_logger
+from preprocessing.preprocess_image import preprocess_binary_image as default_preprocessing
 
 logger = get_logger(__name__)
 
 class Model(ABC):
 
-    def __init__(self, ovms_connector):
+    def __init__(self, model_name, ovms_connector, labels_path):
         self.ovms_connector = ovms_connector
-        self.model_name = None # subtype / model name in AMS
-        self.result_type = None # type class
-        self.labels = None # load_labels
-
-    def load_labels(labels_path: str) -> str:
-        try:
-            with open(labels_path, 'r') as labels_file:
-                data = json.load(labels_file)
-        except Exception as e:
+        self.model_name = model_name
+        self.labels = self.load_labels(labels_path)
+
+    def load_labels(self, labels_path):
+        try:                                                                          
+            with open(labels_path, 'r') as labels_file:                               
+                data = json.load(labels_file)                                         
+                labels = data['outputs'][0]['classes']
```

Comment:
I think we should put here a comment that only one output is currently supported